yew/Cargo.toml

```diff
@@ -141,6 +141,9 @@ agent = ["bincode"]
 yaml = ["serde_yaml"]
 msgpack = ["rmp-serde"]
 cbor = ["serde_cbor"]
+class_macro = [
+	"yew-macro/class_macro"
+]
```

Comment:
I don't think it will be possible to make a feature flag neither having both behavior working at the same time. Well, we CAN but we need to duplicated most of the tests.

Reply:
Definitely don't want a feature flag for this.
Feature flags are additive across the entire dependency tree of an app (up and down) so if a library activates this feature it can be quite problematic.

Reply:
Yeah I already deleted it because it was kinda bad.